### PR TITLE
Link Texts Updated

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :chains: Related Resources

https://chefio.atlassian.net/browse/CHEF-3205

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/node_integrations.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Access the _Node Integrations_ page from the **Settings** tab.
 
 Set up Chef Automate to detect and scan the nodes in your AWS EC2 account by providing your AWS Credentials and creating an _AWS EC2 Node Manager_ from the **Node Credentials** page in the **Settings** tab. Chef Automate requires your information to detect the nodes in your AWS EC2 account. Chef Automate creates a node reference for each EC2 instance in your account and collects all of the tags associated with each instance.
 
-InSpec 2+ supports running scan jobs against your AWS account configuration, such as CloudWatch or IAM, [see more here](https://docs.chef.io/inspec/resources/#aws). Set up Chef Automate to run these scan jobs by providing your AWS Credentials and creating an _AWS API Node Manager_ in the **Node Integrations** page in the **Settings**  tab.
+InSpec 2+ supports running scan jobs against your AWS account configuration, such as [CloudWatch or IAM](https://docs.chef.io/inspec/resources/#aws). Set up Chef Automate to run these scan jobs by providing your AWS Credentials and creating an _AWS API Node Manager_ in the **Node Integrations** page in the **Settings**  tab.
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/node_integrations.md:26]: InSpec should be Chef InSpec, and no one runs less than InSpec 3 (or they shouldn't). so you can remove the `2+`. This text isn't quite right either. It should be "Chef InSpec has [resources](link) you can use to run scan jobs against AWS services in your account, such as CloudWatch or IAM."

**Review by IanMadd — CHANGES_REQUESTED** [review]
Here's some feedback

You also have several links still formatted like this --> `Click [here]` and `Click here to know more [details]` and `[Click here to know how to upgrade to Chef Automate 4.x]`

File: components/docs-chef-io/content/automate/ha_aws_deploy_steps.md [hunk 2]
```diff
@@ -69,7 +69,8 @@ Run the following steps on Bastion Host Machine:
     chef-automate config gen config.toml
     ```
 
-    Click [here](/automate/ha_config_gen) to know more about generating config
+    To know more about on how to generate config, refer to the [Automate HA Config Generation](/automate/ha_config_gen) page.
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/ha_aws_deploy_steps.md:72]: Fixed the changes

File: components/docs-chef-io/content/automate/ha_aws_deploy_steps.md [hunk 3]
```diff
@@ -180,9 +182,9 @@ Check if Chef Automate UI is accessible by going to (Domain used for Chef Automa
 
 After successful deployment, proceed with the following:
 
-   1. Create user and orgs, Click [here](/automate/ha_node_bootstraping/#create-users-and-organization) to learn more about user and org creation
-   1. Workstation setup, Click [here](/automate/ha_node_bootstraping/#workstation-setup) to learn more about workstation setup
-   1. Node bootstrapping, Click [here](/automate/ha_node_bootstraping/#bootstraping-a-node) to learn more about node bootstrapping.
+   1. To know moe about the user and organization creation, check the [Create Users and Organization](/automate/ha_node_bootstraping/#create-users-and-organization) section in Node Bootstrapping page.
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/ha_aws_deploy_steps.md:185]: Fixed the Changes

File: components/docs-chef-io/content/automate/ha_backup_restore_aws_efs.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ A shared file system is always required to create **OpenSearch** snapshots. To r
 
 ### Setting up the backup configuration
 
-- Create an EFS file system, please refer sample steps [here](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html)
+- Create an EFS file system, please refer sample steps from [Create your Amazon EFS file system](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html) page.
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/ha_backup_restore_aws_efs.md:32]: Fixed

File: components/docs-chef-io/content/automate/ha_cert_rotation.md [hunk 5]
```diff
@@ -23,6 +23,9 @@ The certificate rotation is also required when the key for a node, client, or CA
 
 ## Prerequisites
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/ha_cert_rotation.md:26]: Fixed

File: components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md [hunk 6]
```diff
@@ -134,9 +135,9 @@ You can also generate a configuration file using the `init-config` subcommand.
 
     Check if Chef Automate UI is accessible by going to (Domain used for Chef Automate) [https://chefautomate.example.com](https://chefautomate.example.com).
     After successful deployment, proceed with following...
-      1. Create user and orgs, Click [here](/automate/ha_node_bootstraping/#create-users-and-organization) to learn more about user and org creation
-      1. Workstation setup, Click [here](/automate/ha_node_bootstraping/#workstation-setup) to learn more about workstation setup
-      1. Node bootstrapping, Click [here](/automate/ha_node_bootstraping/#bootstraping-a-node) to learn more about node bootstrapping.
+      1. To know moe about the user and organization creation, check the [Create Users and Organization](/automate/ha_node_bootstraping/#create-users-and-organization) section in Node Bootstrapping page.
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md:138]: Fixed

File: components/docs-chef-io/content/automate/large_compliance_report.md [hunk 7]
```diff
@@ -61,7 +61,7 @@ Here is a benchmark test summary report run on
 
 MinIO is a High-Performance Object Storage released under GNU Affero General Public License v3.0. It is API compatible with the Amazon S3 cloud storage service. MinIO is the only object storage suite available on public cloud, every Kubernetes distribution, the private cloud, and the edge.
 
-Click [here](https://min.io) to learn more about MinIO and [set up](https://docs.min.io/minio/baremetal/) the MinIO on a baremetal server.
+For more information on how to set up MinIO in baremetal server, see [MinIO](https://min.io) and how to [set up the MinIO](https://docs.min.io/minio/baremetal/).
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/large_compliance_report.md:64]: Fixed the changes

File: components/docs-chef-io/content/automate/managed_services.md [hunk 8]
```diff
@@ -16,7 +16,7 @@ gh_repo = "automate"
 
 ### PostgreSQL Setup
 
-- Setup PostgreSQL RDS DB instance in AWS. Click [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.PostgreSQL.html) to know more.
+- Setup [PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.PostgreSQL.html) RDS DB instance in AWS.
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/managed_services.md:19]: Fixed

File: components/docs-chef-io/content/automate/managed_services.md [hunk 9]
```diff
@@ -141,7 +146,7 @@ The steps to enable backup and restore in OpenSearch S3 is:
 
       {{< figure src="/images/automate/managed_services_os_dasboard_permission.png" alt="Managed Service OS Dashboard Permission">}}
 
-{{< note >}} To access the default installation of OpenSearch Dashboards for a domain that resides within a VPC, you must have access to the VPC. This process varies by network configuration but likely involves connecting to a VPN or managed network or using a proxy server or transit gateway. Click [here](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/vpc.html#vpc-security) to know more.
+{{< note >}} To access the default installation of OpenSearch Dashboards for a domain that resides within a VPC, you must have access to the VPC. This process varies by network configuration but likely involves connecting to a VPN or managed network or using a proxy server or transit gateway. To know more see [Developer Guide](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/vpc.html#vpc-security).
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/managed_services.md:149]: Fixed

File: components/docs-chef-io/content/automate/node_integrations.md (hunk 1 above)
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/node_integrations.md:26]: Fixed

File: components/docs-chef-io/content/automate/ha_config.md [hunk 10]
```diff
@@ -26,7 +26,7 @@ gh_repo = "automate"
 
 {{< warning >}}
 
-- For certificate rotation, don't use `config patch`. Instead, the `cert-rotate` command can be used. To know more about certificate rotation, click [here](/automate/ha_cert_rotation)
+- For certificate rotation, don't use `config patch`. Instead, the `cert-rotate` command can be used. To know more, see [certificate rotation](/automate/ha_cert_rotaion).
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/ha_config.md:29]: I couldn’t get your comment there. Are you telling there that the link is incorrect and I should use another one?

[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/ha_config.md]: FIxed The Changes.

**Review by IanMadd — APPROVED** [review]

**Review by punitmundra — APPROVED** [review]

**Pull request merged**